### DISCUSSION TITLE
What number is ...

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#750] [Galaxy Trivia] [answered]
... one half of one third of one quarter of 360?

*answer in hex*

**Comment by SANKARAMDAS** [comment, marked as answer]
15

Hex number (1 digit)  - F 
Hex signed 2's complement (4 digits) -  000F

**Comment by peterbaccaro** [comment]
One half of one third of one quarter of 360 can be calculated as:

(1/2) x (1/3) x (1/4) x 360 = 15

Therefore, one half of one third of one quarter of 360 is equal to 15 which in hex is represented as **F**.
